src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/SelectVariants.java

```diff
@@ -143,28 +143,15 @@
     private Set<String> sampleExpressions = new LinkedHashSet<>(0);
```

Comment:
This argument takes a mixed list of `.list` files and expressions.  These are evaluated by ListFileUtils.includeMatching where the `.list` files are expanded.  I think we should remove `ListFileUtils`  because it's now redundant.  Some of the code for matching regex in ListFileUtils should be migrated here if it's necessary.